HW0/P4/P4.py

```diff
@@ -23,55 +23,112 @@
 
     # Create 3D axes for plotting
     ax = Axes3D(plt.figure())
-
+    
     #####################
     # Part 1:
     #
     # Load true trajectory and plot it
     # Normally, this data wouldn't be available in the real world
     #####################
+    
+    traj=np.loadtxt("P4_trajectory.txt", delimiter=",");
+    np.reshape(traj,(120,6));
```

Comment:
I reshape to be sure of what I am working with.
If I don't know the number of time step I can do a little algebra with np.size to recover it (since I know that there are 6 columns)